runtime/caml/misc.h

```diff
@@ -199,6 +199,8 @@ CAMLextern void caml_fatal_error (char *, ...)
 #endif
 CAMLnoreturn_end;
 
+#define Is_power_of_2(x) (((x) & ((x) - 1)) == 0)
```

Comment:
This should be protected with `#ifdef CAML_INTERNALS`

Reply:
Updated in 4e084b5721eabeea8d75214bf811d9e0a651b1e5.